eng/CodeChecks.ps1

```diff
@@ -71,7 +76,7 @@ try {
     if ($LastExitCode -ne 0) {
```

Comment:
Why do we need `--ignore-space-at-eol` the generation tools should not be generating different spacing around eol.

Reply:
Is to ignore different line endings that sometimes get generated.

Reply:
We should update our generation tools to make that not an issue but I'm OK with the diff ignoring it for now.